src/ens/services/ens/index.ts

```diff
@@ -0,0 +1,11 @@
+import ENS = require('ez-ens');
```

Comment:
Putting this file in `src/ens/services/ens.ts` should be fine. It's simple enough to not need a directory.